### DISCUSSION TITLE
公子厨看到能评价下吗

### DISCUSSION THREAD

**Discussion by molinFireFly** [share121/inter-knot#1007] [其他]
![mmexport1723293701773](https://github.com/user-attachments/assets/264f8140-e891-4ed3-b0e9-84b56f695d35)
![Screenshot_20240810_204549](https://github.com/user-attachments/assets/08d1cf36-5141-42b6-a29e-9a29344eb483)
我这公子如果刷出来一个四五十分的杯子是不是毕业了（有2万），坐牢一年多还没出双爆水杯

**Comment by SonwNight3089** [comment]
好牛逼的圣遗物，我的面板比你还低。玩万达的话这样的公子已经够用了，该提升香菱练度XD
